addons/project_timesheet_holidays/__init__.py

```diff
@@ -20,23 +19,23 @@ def post_init(env):
         if not company.internal_project_id:
             if not internal_projects_by_company_dict:
                 internal_projects_by_company_read = project.search_read([
-                    ('name', '=', _('Internal')),
+                    ('name', '=', project.env._('Internal')),
```

Comment:
You have `env` as a parameter, you don't need to use `project`.
```suggestion
                    ('name', '=', env._('Internal')),
```